common/lib/xmodule/xmodule/course_module.py

```diff
@@ -264,18 +267,32 @@ class CourseFields(object):
     )
     discussion_blackouts = List(
         display_name=_("Discussion Blackout Dates"),
-        help=_("Enter pairs of dates between which students cannot post to discussion forums. Each pair should be formatted as [\"YYYY-MM-DD\", \"YYYY-MM-DD\"]. To specify times as well as dates, format each pair as [\"YYYY-MM-DDTHH:MM\", \"YYYY-MM-DDTHH:MM\"] (be sure to include the \"T\" between the date and time). An entry defining more than one blackout period might look like this: [[\"2014-09-15\", \"2014-09-21\"], [\"2014-10-01\", \"2014-10-08\"]]"),
+        help=_(
+            "Enter pairs of dates between which students cannot post to discussion forums. Each pair should be "
+            "formatted as [\"YYYY-MM-DD\", \"YYYY-MM-DD\"]. To specify times as well as dates, format each pair "
+            "as [\"YYYY-MM-DDTHH:MM\", \"YYYY-MM-DDTHH:MM\"] (be sure to include the \"T\" between the date and "
+            "time). An entry defining more than one blackout period might look like this: "
+            "[[\"2014-09-15\", \"2014-09-21\"], [\"2014-10-01\", \"2014-10-08\"]]"
```

Comment:
Let's just try it with this string and the string that's after it (`"Enter discussion categories in the following format: \"CategoryName\": "...`) and test to make sure it scrapes properly. I want to double check that's correct. I don't understand why the engineer would have otherwise done this weird escaping thing.